src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -866,3 +866,16 @@ export type Chunks = {
   remark?: string;
   rule_id?: string;
 };
+
+export enum ResolvementStatus {
```

Comment:
ResolutionStatus might be a better name. Resolvement isn't a standard word.

Reply:
Thanks!

![tenor](https://user-images.githubusercontent.com/9060071/75324263-d881ec80-5876-11ea-95c4-ae238bd7ed3a.gif)